python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -514,18 +679,26 @@ def predict(self, data, batch_size=32, quantize=False):
             return yhat
         else:
             if not self.fitted:
-                from bigdl.nano.utils.log4Error import invalidInputError
                 invalidInputError(False,
                                   "You must call fit or restore first before calling predict!")
             if quantize:
                 yhat = _pytorch_fashion_inference(model=self.pytorch_int8,
                                                   input_data=data,
                                                   batch_size=batch_size)
             else:
-                self.internal.eval()
-                yhat = _pytorch_fashion_inference(model=self.internal,
-                                                  input_data=data,
-                                                  batch_size=batch_size)
+                if acceleration is False:
+                    self.internal.eval()
+                    yhat = _pytorch_fashion_inference(model=self.internal,
+                                                      input_data=data,
+                                                      batch_size=batch_size)
+                else:
+                    invalidInputError(self.optim_model is not None,
+                                      "There is not optimized model, please call "
+                                      ".optimize() first.")
```

Comment:
I think a better way here is to just states the behavior in api doc rather than an error, and in the mean time the prediction is fallback to original model. This will save a lot of work in ut's changing and, in other word, more friendly to users who are familiar with the old API.

Reply:
I agree. Good suggestion and I will adopt this way.